package.json

```diff
@@ -7,22 +7,22 @@
   "scripts": {
     "eslint": "eslint packages/ --ext .ts",
     "prettier": "prettier --check './packages/**/*.ts' '!**/*.d.ts'",
-    "clean": "rimraf packages/*/dist bin",
-    "reset": "yarn clean && rimraf .parcel-cache packages/*/node_modules && rimraf node_modules",
-    "package": "yarn clean && yarn build && node ./scripts/package",
-    "release": "yarn node ./scripts/release",
+    "clean": "mkdirp packages/base/dist && rimraf packages/*/dist bin etc/jest/.test-data",
+    "reset": "yarn clean && mkdirp packages/base/node_modules && rimraf .parcel-cache packages/*/node_modules node_modules .yarn/cache .yarn/install-state.gz .yarn/unplugged .pnp.cjs .pnp.loader.mjs",
```

Comment:
I think we shouldn't keep anything related to yarn PnP till we give it another go

Reply:
Everything works for PnP except pkg so we can use PnP to sort out package deps. In fact we can put the package in PnP when developping we just need node_modules to create the binary.

Reply:
Please let not do this as it will create a mess of configs and be so hard to debug in case we have issues.